b2sdk/v2/sync.py

```diff
@@ -0,0 +1,13 @@
+######################################################################
+#
+# File: b2sdk/v2/sync.py
+#
+# Copyright 2022 Backblaze Inc. All Rights Reserved.
+#
+# License https://www.backblaze.com/using_b2_code.html
+#
+######################################################################
+
+from b2sdk._v3 import B2Path
+
+B2SyncPath = B2Path
```

Comment:
it's not enough, I think you need to:
```python
class B2SyncPath(B2Path):
    pass
```
in order to make `__str__` consistent with what used to be public in v2. `__str__` is kind of part of the public interface, keeping it here doesn't cost much.

Reply:
Akselandr Goncharov

Sorry, again, __str__ remains the same (class name isn't used in __str__). The __repr__ does change however, but is it a part of public interface? I think noone should rely on repr()

Paweł Polewicz
repr is NOT a part of the public interface